packages/db/index.js

```diff
@@ -174,13 +175,13 @@ export class DBClient {
   /**
    * Create a user tag
    * @param {string} userId
-   * @param {Object} [tag]
-   * @param {string} [tag.tag]
-   * @param {string} [tag.value]
-   * @param {string} [tag.reason]
+   * @param {import('./db-client-types').UserTagInput?} tag
    * @returns {Promise<boolean>}
    */
-  async createUserTag (userId, tag = {}) {
+  async createUserTag (userId, tag) {
+    if (!tag || !tag.tag) {
```

Comment:
```suggestion
    if (tag?.tag) {
```

Reply:
Ah yes, but with `!`. I'll make this change.

Reply:
ups yeah :)